### ISSUE TITLE
faker.image.avatar() console.logs "Deprecation Warning: faker.random.number is now located in faker.datatype.number"

### ISSUE THREAD

**Issue by Lateralus58** [faker-js/faker#261]
### Describe the bug

Any call to `faker.image.avatar()` or `faker.internet.avatar()` result in a deprecation warning being outputted to the console:

`
Deprecation Warning: faker.random.number is now located in faker.datatype.number
`

### Reproduction

Simply import faker and call `faker.image.avatar()` or `faker.internet.avatar()`

```
const faker = require('@faker-js/faker')

faker.image.avatar()
faker.internet.avatar()
```

### Additional Info

Version: `"@faker-js/faker": "^6.0.0-alpha.3"`

**Comment by Shinigami92** [comment]
6.0.0-alpha.4 is not released yet
This is already fixed on main branch

**Comment by Lateralus58** [comment]
> 6.0.0-alpha.4 is not released yet This is already fixed on main branch

I realized that when I forked and built. I will update my local .3 version with the fix until 6.0.0-alpha.4 is released.

Thank you for the prompt response

**Comment by Lateralus58** [comment]
I realized that when I forked and built. I will update my local .3 version with the fix until 6.0.0-alpha.4 is released.

Thank you for the prompt response
